src/frontends/pytorch/src/op/slice.cpp

```diff
@@ -74,6 +74,14 @@ OutputVector translate_slice(const NodeContext& context) {
     return {context.mark_node(std::make_shared<v8::Slice>(context.get_input(0), start, end, step, dim))};
 };
 
+OutputVector translate_slice(const NodeContext& context) {
+    return translate_slice_common(context, context.get_input_size());
+};
+
+OutputVector translate_slice_fx(const NodeContext& context) {
+    return translate_slice_common(context, 5);
```

Comment:
Why exactly you do that? How many inputs are really in FX? At least add comment here

Reply:
Added comments